integrade/utils.py

```diff
@@ -21,3 +25,20 @@ def gen_password(length=20):
 def uuid4():
     """Provide unique string identifiers."""
     return str(uuid.uuid4())
+
+
+def _num_valid_cloud_accounts():
+    cfg = config.get_config()
```

Comment:
Docstring?

Reply:
decided to ditch this with switch to list of ARNs